packages/camera/camera_android/example/lib/camera_controller.dart

```diff
@@ -112,6 +117,7 @@ class CameraValue {
     DeviceOrientation? recordingOrientation,
     bool? isPreviewPaused,
     DeviceOrientation? previewPauseOrientation,
+    bool clearNullOrientationsFlag = false,
```

Comment:
When do we want to pass false?

Reply:
I took a different approach, so closing this.